module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2449,22 +2470,37 @@ protected function mapRenewalBlockReason($reason, $itype)
     }
 
     /**
-     * Return a location for a Koha item
+     * Return a location (branch or shelving) for a Koha item
      *
      * @param array $item Item
      *
      * @return string
      */
     protected function getItemLocationName($item)
     {
-        $libraryId = (!$this->useHomeLibrary && null !== $item['holding_library_id'])
-            ? $item['holding_library_id'] : $item['home_library_id'];
-        $name = $this->translateLocation($libraryId);
-        if ($name === $libraryId) {
-            $libraries = $this->getLibraries();
-            $name = isset($libraries[$libraryId])
-                ? $libraries[$libraryId]['name'] : $libraryId;
+        $name = '';
```

Comment:
Now that your switch is set up in a way that every branch is guaranteed to set `$name` to a value, you don't need to initialize it here.